game/resource/English/modifier/items/modifier_meteor_hammer.txt

```diff
@@ -0,0 +1,8 @@
+//=============================================================================
+// Meteor Hammer
+//=============================================================================
+"DOTA_Tooltip_modifier_item_meteor_hammer_damage_over_time"              "Meteor Hammer Burn"
+"DOTA_Tooltip_modifier_item_meteor_hammer_damage_over_time_Description"  "Does magical damage over time for a certain amount of time"
```

Comment:
The vanilla modifier's tooltip is simply "Taking damage over time." You really don't need to say "for a certain amount of time". If you want to 1-up Valve, you could have this be "Taking %dMODIFIER_PROPERTY_TOOLTIP% magic damage per second." And then add `MODIFIER_PROPERTY_TOOLTIP` to the modifier's `DeclareFunctions` and add a `OnTooltip` method that returns the damage per second.